articles/hooks/guides/create-delete-hooks-using-cli.md

```diff
@@ -6,20 +6,28 @@ topics:
     - cli
 contentType: how-to
 useCase: extensibility-hooks
+v2: true
 ---
+# Create/Delete Hooks Using the Command-Line Interface
 
-# Create/Delete Hooks
+The Auth0 Command-Line Interface (CLI) allows you to create and delete Hooks associated with specific extensibility points within the Auth0 platform.
 
 ::: warning
 Tenants created after **July 16, 2018** will not have access to the underlying Webtask Sandbox via the Webtask CLI. Please contact [Auth0](https://auth0.com/?contact=true) to request access.
 :::
 
+## Set up the CLI
+
+You can find instructions for installing and configuring the Webtask CLI in the [Dashboard > Webtask page](${manage_url}/#/account/webtasks). 
+
+The `wt-cli` package also includes the `auth0` binary, allowing you to use the Auth0 CLI.
+
+![Install Webtasks Instructions](/media/articles/hooks/mgmt-dashboard-webtasks.png)
+
 ::: note
 The Auth0 CLI examples use `auth0-profile` as the name of the profile. This is the same profile name used when installing `wt-cli`, and you can obtain it from *Step 2* of the instructions set located on [Auth0 Management Dashboard's Webtask page](${manage_url}/#/account/webtasks).
 :::
 
-Using the Auth0 CLI, you can create new Hooks, as well as manage or delete existing Hooks. You can also gather real-time data about your Hooks.
-
 ## Create a New Hook
```

Comment:
Sentence case on all H2s I think is the standard. I could be wrong, but that's what I've been doing. Title case for H1s only.